src/foam/nanos/auth/services.jrl

```diff
@@ -194,9 +197,11 @@ p({
   "name":"languageDAO",
   "lazy":false,
   "serve":true,
+  "authenticate": false,
```

Comment:
I'm not 100% on this change, but I don't see why languages is any different than countrydao/regiondao/etc

Reply:
in any case it was erroring out for my unauthed user due to unable to find languageDAO

Reply:
@adamvy The languageDAO nspecs requires a GlobalReadAuthorizer. 
replace the authenticate:false with
       .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("language"))